src/DetailsView/components/assessment-test-view.tsx

```diff
@@ -60,14 +63,21 @@ export const AssessmentTestView = NamedFC<AssessmentTestViewProps>(
             assessmentNavState.selectedTestType,
             assessmentData,
         );
-        return (
-            <>
-                <ScanIncompleteWarning
+
+        const renderReflowAssessmentView = (): JSX.Element => {
```

Comment:
FYI, these don't have to be functions. You can store the components as constants themselves.

i.e. something like this:

```
        const reflowAssessmentView = (
            <ReflowAssessmentView
                deps={deps}
                assessmentNavState={assessmentNavState}
                assessmentData={assessmentData}
                currentTarget={currentTarget}
                prevTarget={prevTarget}
                assessmentTestResult={assessmentTestResult}
            />
        );
```